news/2018-05-03-beatmap-spotlights-march-2018.md

```diff
@@ -0,0 +1,117 @@
+
+[![](/wiki/shared/news/2018-05-03-beatmap-spotlights-march-2018/happy-halloween.jpg)](http://osu.ppy.sh/beatmapsets/692433)
+
+Halloween comes early this year as [Loneight](https://osu.ppy.sh/users/663131) returns with a set of yuikonnu & ayaponzu*'s "Happy Halloween". The map shows off the crazy feel of this song very well, switching between swing beat and normal beats fairly often, and keeping you on your toes for the various SV changes going on. Coupled with wonderful improvised hitsounding, this map definitely brings together a fun experience.
+
+[![](/wiki/shared/news/2018-05-03-beatmap-spotlights-march-2018/cosmic-love.jpg)](http://osu.ppy.sh/beatmapsets/696222)
+
+Fans of western electronic music won't be left out this month, as the team of [PandaHero](https://osu.ppy.sh/users/1233255), [La Cataline](https://osu.ppy.sh/users/301279), and [Mirash](https://osu.ppy.sh/users/2841009) bring forward their mapset of Florence + The Machine's "Cosmic Love". The top difficulty shows off great usage of slow sliders, while the rest of the spread keeps a varied and enjoyable experience for players of all skill levels.
+
+[![](/wiki/shared/news/2018-05-03-beatmap-spotlights-march-2018/on-the-backs-of-angels.jpg)](http://osu.ppy.sh/beatmapsets/707720)
+
+With his latest ranked map of Dream Theater's "On The Backs Of Angels", [LMT](https://osu.ppy.sh/users/7262798) manages to find a great balance between playability and capturing the complexity of a nearly 9 minute long progressive metal song.
+If you're looking for a map that challenges consistency and finger control, you should definitely give this a try!
+
+[![](/wiki/shared/news/2018-05-03-beatmap-spotlights-march-2018/tokei-no-heya.jpg)](http://osu.ppy.sh/beatmapsets/508947)
+
+And finally, we have one more beatmap from a_hisa - Tokei no Heya to Seishin Sekai mapped by [Asahina Momoko](https://osu.ppy.sh/users/3650145) and a team of experienced guest mappers! a_hisa is mainly well known for creating spectacular Spiritual Core music and Tokei no Heya to Seishin Sekai is no exception. Complemented by exquisite melody and beats, the song is as engaging as it is emotional.
+
+Momoko has done a great job here on capturing this song through his difficulty, featuring peculiar patterns arranged in a clean and polished fashion which perfectly resonates with every single detail of the song. The rest of Momoko's mapset is not to be neglected either - they are no less expressive when compared to the top difficulty and is definitely worth playing as well!
+
+[![](/wiki/share/news/2018-05-03-beatmap-spotlights-march-2018/dum-surfer.jpg)](http://osu.ppy.sh/beatmapsets/716193)
```

Comment:
The image isn't showing up because shared in the url is missing a d

Reply:
How in the earth did this happen, i wonder. lol Can only fix when I get home, approx ~9 more hours